src/containers/Application/ApplicationPage.tsx

```diff
@@ -140,8 +80,8 @@ const ApplicationPage: React.FC<ApplicationProps> = ({ structure }) => {
         <Grid.Column width={4}>
           <ProgressBarNEW
             structure={fullStructure}
-            requestRevalidation={requestRevalidation}
-            strictSectionPage={strictSectionPage}
+            requestRevalidation={requestRevalidation as MethodRevalidate}
```

Comment:
hmm it looks like types can be improved, but that's being picky. 
not required change, but if we weren't so pressed for time would make props for ApplicationPage props requestRevalidation not optional and strictSection as SectionAndPage| null

then don't have to cast `as` in the page

Reply:
Yeah, the problem is we're using the same type (ApplicationProps) for both Wrappers, and the top-most one doesn't have those last two properties to pass down.